src/components/menu/menu.tsx

```diff
@@ -107,6 +122,26 @@ const MenuButton = polymorphicComponent<'button', MenuButtonProps>(function Menu
     )
 })
 
+//
+// MenuContextMenuTrigger
+//
+const MenuContextMenuTrigger = polymorphicComponent<'div', unknown>(function MenuContextMenuTrigger(
```

Comment:
@frankieyan 

This is a bit of a nit-pick, but I'm having trouble parsing this name in my mind.

What about `ContextMenuTrigger`?

Reply:
I was torn on this one, as the other components here have `Menu` as a prefix, but let's go with your suggestion 👍